linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/rule.yml

```diff
@@ -1,27 +1,27 @@
-documentation_complete: true
+{{% if product in ["sle12", "sle15"] %}}
+{{% set xwindows_packages = ['xorg-x11-server', 'xorg-x11-server-extra', 'xorg-x11-server-Xvfb', 'xwayland'] %}}
```

Comment:
Also this variable set is not needed

Reply:
Thanks :bow: should be handled in [aeb38eb](https://github.com/ComplianceAsCode/content/pull/12853/commits/aeb38eb0aa62316ccd418bafe7b1009e688f0cf0)